t/plugin/proxy-rewrite3.t

```diff
@@ -619,3 +622,50 @@ GET /test/plugin/proxy/rewrite HTTP/1.1
     }
 --- response_body
 /plugin_proxy_rewrite?a=c
+
+
+
+=== TEST 27: set route
```

Comment:
Is it a duplicate of test 19?

Reply:
Just want prepare configuration for Test 28.

Reply:
Aren't Test 28 and Test 20 duplicates?

Reply:
Ok, I will merge them. At first, I just want one test focus on one function pont.

Reply:
Fixed.